tasks/sep/006-0-sc-guardian-updates/VALIDATION.md

```diff
@@ -0,0 +1,79 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xa87675ebb9501C7baE8570a431c108C1577478Fa` (The Security Council Safe)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xa87675ebb9501C7baE8570a431c108C1577478Fa)
+
+Enables the simulation by setting the threshold to 1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+## State Changes
+
+### `0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820` (`LivenessGuard`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820) (not verified)
```

Comment:
> can we verify it?

I've had difficulty with verification. If we want it to block this ceremony then I will either need help from Solomon who has more experience with source verification, or it will need to be postponed. I did call it out as [scope I wanted to cut](https://oplabs-pbc.slack.com/archives/C06T3GFN8NB/p1713450842202579?thread_ts=1713441989.907499&cid=C06T3GFN8NB) and didn't get any objections. :)

Reply:
> I also wanted to verify that all storage slots in this contract are correctly initialized, including the fallback address, proxy admin (if upgradable). 

All return values in the Liveness Module and Guard are validated here:

https://github.com/ethereum-optimism/superchain-ops/blob/9e4ae9ec075409837557e244f2fe9552bc2df4bc/tasks/sep/006-0-sc-guardian-updates/SignFromJson.s.sol#L379-L409

This validation approach was agreed upon with @mds1. It would be good to have his eyes on that code.